LEGO1/legoentity.h

```diff
@@ -31,7 +32,16 @@ class LegoEntity : public MxEntity
     return !strcmp(name, LegoEntity::ClassName()) || MxEntity::IsA(name);
   }
 
+  virtual void vtable18(); // vtable+0x18
   virtual void Destroy() override; // vtable+0x1c
```

Comment:
This is a new virtual function introduced in `LegoEntity`, so we have to remove the `override` keyword.